### ISSUE TITLE
Failed to resolve: im.ene.toro3:toro-core:3.5.0

### ISSUE THREAD

**Issue by DummyCo** [eneim/toro#336]
I'm afraid there is some kind of distribution problem with 3.5.0:
`Failed to resolve: im.ene.toro3:toro-core:3.5.0`

**Comment by eneim** [comment]
Hm I think toro-core is module name but gradle should be im.ene.toro3:toro:3.5.0 only.

**Comment by DummyCo** [comment]
@eneim, yep, that's it toro only:
```
def toro = [:]
toro.core = "im.ene.toro3:toro:$versions.toro"
toro.ext_exoplayer = "im.ene.toro3:toro-ext-exoplayer:$versions.toro"
deps.toro = toro
```
maybe there is some kind of misconfigured internal module dependency?

**Comment by eneim** [comment]
I see. Let me double check this after coming home >.<

**Comment by DummyCo** [comment]
@eneim quite sure problem is somewhere around toro-ext-exoplayer toro-core dependency on second line:
```
+--- im.ene.toro3:toro-ext-exoplayer:3.5.0
|    +--- im.ene.toro3:toro-core:3.5.0 FAILED
|    +--- com.google.android.exoplayer:exoplayer:2.7.3
|    |    +--- com.google.android.exoplayer:exoplayer-core:2.7.3 (*)
|    |    +--- com.google.android.exoplayer:exoplayer-dash:2.7.3 (*)
|    |    +--- com.google.android.exoplayer:exoplayer-hls:2.7.3
|    |    |    \--- com.google.android.exoplayer:exoplayer-core:2.7.3 (*)
|    |    +--- com.google.android.exoplayer:exoplayer-smoothstreaming:2.7.3
|    |    |    \--- com.google.android.exoplayer:exoplayer-core:2.7.3 (*)
|    |    \--- com.google.android.exoplayer:exoplayer-ui:2.7.3 (*)
|    +--- com.android.support:recyclerview-v7:27.1.1 (*)
|    \--- com.android.support:support-annotations:27.1.1
```
Thank you!

**Comment by eneim** [comment]
Got my hand on PC now, hope it is not caused by the changed in release script and Gradle 4.6. Will take a look now.

**Comment by eneim** [comment]
@DummyCo A quick comment is the issue looks the same as [this one](https://stackoverflow.com/questions/37250675/how-to-setup-dependencies-in-a-multi-module-android-gradle-project-with-interdep) (but not identical). So the pom file generated for toro-ext-exoplayer will use toro-core as artifactId for its dependency, but then it is not the one to resolve from maven.

Still looking for a good solution other than renaming the modules ...

**Comment by eneim** [comment]
@DummyCo I guess I fix the issue by replacing it by 3.5.2 artifacts (no changes in functionality ...). Please update your gradle and see if it works.

**Comment by DummyCo** [comment]
@eneim yep, it's working now.
Thank you for first-class support as always :1st_place_medal:
